docs/guides/user/docs/advanced/offload-s3.md

```diff
@@ -0,0 +1,38 @@
+[img_select_events]: media/img/select_events.png
+[img_confirm_events]: media/img/confirm_events.png
+[img_offload_task]: media/img/offload_task.png
+[img_offload_s3_1]: media/img/offload_s3_1.png
+[img_offload_s3_2]: media/img/offload_s3_2.png
```

Comment:
The images are showing up as broken links to me. I did "mkdocs build" and opened it in a browser.

Reply:
Images went missing in the numerous rebases.  Good thing I still have the base codebase!